src/Microsoft.ML.Data/DataLoadSave/CompositeReaderEstimator.cs

```diff
@@ -8,13 +8,13 @@ namespace Microsoft.ML.Data
     /// An estimator class for composite data reader.
```

Comment:
```suggestion
    /// An estimator class for composite data loader.
```
So many `reader`s. Is it possible to do a global text replacement?